functions/backup.bash

```diff
@@ -160,10 +160,11 @@ create_amanda_config() {
   echo -n "$(timestamp) [openHABian] Creating Amanda filesystem... "
   if ! cond_redirect mkdir -p "$configDir" "$databaseDir" "$logDir" "$indexDir"; then echo "FAILED (create directories)"; return 1; fi
   if ! cond_redirect touch "${configDir}/tapelist"; then echo "FAILED (touch tapelist)"; return 1; fi
-  ip=$(dig +short "$HOSTNAME")
+  # shellcheck disable=SC2154
+  ip=$(dig +short "$hostname")
```

Comment:
Will this work if the hostname that the machine currently has and its hostname that the user wants to set are different?

Reply:
$hostname is taken (early on start) from openhabian.conf so I think yes

Reply:
But does it take effect before a reboot? If it did then this entire PR probably wouldn't be necessary.

Reply:
The point is whatever is in $hostname is e.g. written to disklist. 
It does not matter when it takes effect.

Reply:
But it might in this case because wouldn't dig be expecting the current hostname when it was looking for an IP not the hostname that the user has set but that hasn't taken effect. Do you get what I mean?

Reply:
I get it. Frankly I'm not sure when in the install process it takes effect but amanda install is always *after* a reboot so I hoped there it did for sure. But we should test.

Reply:
Then again HOSTNAME has worked before and there's no need to change it. So I just reverted this change.